gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/FsBenchmark.java

```diff
@@ -235,16 +318,17 @@ private void benchmarkRead(
     // Verify that all threads completed without errors
     futures.forEach(Futures::getUnchecked);
 
-    printTimeStats("Read call time", readCallTimeNsList);
-    printSizeStats("Read call data", readCallBytesList);
-    printThroughputStats("Read call throughput", readCallTimeNsList, readCallBytesList);
+    printTimeStats(operation + " call time", readCallTimeNsList);
```

Comment:
please rename `read*` references to `operation*` or have explicit vars for read and write